prototypes/basic/app/views/review.html

```diff
@@ -2,14 +2,67 @@
 
 {% block pageTitle %} {{ serviceName }} – GOV.UK Prototype Kit {% endblock %}
 {% block beforeContent %}
-{{ govukBackLink({ text: "Back", href: "javascript:window.history.back()" }) }}
+{{
+    govukBreadcrumbs({
+      items: [
+        {
+          text: "Home",
+          href: "/"
+        },
+        {
+            text: "Upload spreadsheet",
+            href: "/upload"
+        },
+        {
+          text: "Select sheet",
+          href: "/select_sheet"
+        },
+        {
+            text: "Select header row",
+            href: "/select_header_row"
+        },
+        {
+            text: "Select footer row",
+            href: "/select_footer_row"
+        },
+        {
+            text: "Identify columns",
+            href: "/mapping"
+        },
+        {
+            text: "Review data",
+            href: "/review"
+        }
+      ]
+    })
+  }}
 {% endblock %}
 
 {% block content %}
+{% set results = importerMappedData(data) %}
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">Review your data</h1>
 
+        <p class="govuk-body">
+            During import there were <strong>{{results.errorCount}}</strong> errors and <strong>{{results.warningCount}}</strong> warnings.
+        </p>
+
+        <p class="govuk-body">
+        The uploaded data contains the following data:
+
+            <ul class="govuk-list govuk-list--bullet">
+                <li><strong>{{ results.totalCount }}</strong> rows of data</li>
+                <li>a total salary commitment from the 'Salary' column of <strong>{{ data_sum(data, 'Salary') | currency }}</strong>.</li>
+                <li>an average salary from the 'Salary' column of <strong>{{ data_avg(data, 'Salary') | currency }}</strong>.</li>
+            </ul>
+        </p>
+
+        <p class="govuk-body">
+            If the details above are correct you can continue to submit your return by clicking the 'Submit' button below.
+        </p>
+
```

Comment:
I think there needs to be a back link or some indication of what to do if you're not happy with the review.

Reply:
Added more text in 7682071